google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/prepare/datacatalog_tag_factory.py

```diff
@@ -78,4 +78,12 @@ def make_tag_for_topic(self, tag_template, topic_metadata):
             tag.fields[template_fields.max_compaction_lag_as_text.
                        name].string_value = topic_metadata.get(
                            MetadataConstants.MAX_COMPACTION_LAG_TEXT)
+        key_schema = topic_metadata.get(MetadataConstants.TOPIC_KEY_SCHEMA)
+        if key_schema:
+            tag.fields[
+                template_fields.key_schema.name].string_value = key_schema
+        value_schema = topic_metadata.get(MetadataConstants.TOPIC_VALUE_SCHEMA)
+        if value_schema:
```

Comment:
Evaluate if it makes sense to also add the schema to the `entry.schema.columns`, so users are able to tag the topic schema.

Reply:
I think it is a really important feature to have - I will try and open a new PR for it in the next couple of days

Reply:
sounds good